iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateBranchFeeders.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.ConnectablePosition;
+import com.powsybl.iidm.network.extensions.ConnectablePositionAdder;
+
+import java.util.Objects;
+
+/**
+ * This class allows to add a new branch's feeders on existing busbar sections. The voltage level containing the
+ * busbar section should be described in node/breaker topology. The branch is created and connected to
+ * the busbar sections with a breaker and a closed disconnector each. The branch is also connected to all
+ * the parallel busbar sections, if any, with an open disconnector.
+ *
+ * @author Miora Vedelago <miora.ralambotiana at rte-france.com>
+ */
+public class CreateBranchFeeders extends AbstractCreateConnectableFeeders {
+
+    private final BranchAdder<?> branchAdder;
+    private final String bbsId1;
+    private final String bbsId2;
+    private final int positionOrder1;
+    private final int positionOrder2;
+    private final ConnectablePosition.Direction direction1;
+    private final ConnectablePosition.Direction direction2;
+
+    CreateBranchFeeders(BranchAdder<?> branchAdder, String bbsId1, String bbsId2, Integer positionOrder1, Integer positionOrder2,
+                        ConnectablePosition.Direction direction1, ConnectablePosition.Direction direction2) {
+        super(1, 2);
+        this.branchAdder = Objects.requireNonNull(branchAdder);
+        this.bbsId1 = Objects.requireNonNull(bbsId1);
+        this.bbsId2 = Objects.requireNonNull(bbsId2);
+        this.positionOrder1 = Objects.requireNonNull(positionOrder1);
```

Comment:
The constructor takes an `Integer`, but it cannot be null, as in `CreateFeederBay` in fact. It's because of the builder I suppose, but shouldn't the `Objects.requireNonNull` be in the builder then?

Reply:
Yes, it is because of the builder. I put all the checks for homogenization in the constructor (none of the builders have checks so far) but this can be changed, I don't really have any preference. I would prefer however have all the checks in one place, would you rather have them all in the builder (when calling `build()`) or in the constructor?

Reply:
In fact I was just thinking about a user which would use `CreateBranchFeeders` instead of the builder, then it's strange to ask for an `Integer`. But I missed the fact that such a user shouldn't exist as it's package-private.